pkg/pool-linear/contracts/LinearPool.sol

```diff
@@ -46,36 +55,53 @@ contract LinearPool is BasePool, IGeneralPool, LinearMath, IRateProvider {
 
     uint256 private constant _TOTAL_TOKENS = 3; // Main token, wrapped token, BPT
 
-    // Linear Pools don't lock any BPT, since they fully support having zero main and wrapped token balances
-    uint256 private constant _LINEAR_MINIMUM_BPT = 0;
-    uint256 private constant _MAX_TOKEN_BALANCE = 2**(112) - 1;
+    // This is the maximum token amount the Vault can hold. In regular operation, the total BPT supply remains constant
+    // and equal to _INITIAL_BPT_SUPPLY, but most of it remains in the Pool, waiting to be exchanged for tokens. The
+    // actual amount of BPT in circulation is the total supply minus the amount held by the Pool, and is known as the
+    // 'virtual supply'.
+    // The only instance in which the total supply might change is if the emergency pause is turned on, enabling an
+    // alternative way to exit the Pool which involves burning BPT. As this is expected to not happen, we optimize for
+    // success by using _INITIAL_BPT_SUPPLY instead of totalSupply(), saving a storage read. This optimization is only
+    // valid if the Pool is never paused: in the case of an emergency that leads to burned tokens, the Pool should not
+    // be unpaused and continue to be used via regular means.
```

Comment:
```suggestion
    // valid if the Pool is never paused: in case of an emergency that leads to burned tokens, the Pool should not
    // be used after the buffer period expires and it automatically 'unpauses'.
```